ProtoSchool/protoschool.github.io
# Fix: prerender redirect pages

Closes #520 
Blocked by #521 since we needed the latest SEO changes.

To test this PR:

```
npm run build
npm run cy:serve
```

Check `localhost:3000/chapters` redirects to `/events`. Before this PR it would return 404.
